src/lib/storage/value_segment.cpp

```diff
@@ -119,7 +119,6 @@ template <typename T>
 void ValueSegment<T>::resize(const size_t size) {
   DebugAssert(size > _values.size() && size <= _values.capacity(),
               "ValueSegments should not be shrunk or resized beyond their original capacity");
-  access_counter[SegmentAccessCounter::AccessType::Sequential] += _values.size();
```

Comment:
@mflueggen Noticed this while working on something else. As the vectors are preallocated, we never move data. In fact, multi-threading on ValueSegment would be completely broken if the data was ever touched as part of resize. As such, this counter increment looks incorrect.

Reply:
True.